website/docs/r/drds_instance.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_drds_instance"
+sidebar_current: "docs-alicloud-resource-drds-instance"
+description: |-
+  Provides an DRDS instance resource.
+---
+
+# alicloud\_drds\_instance
+
+Distributed Relational Database Service (DRDS) is a lightweight (stateless), flexible, stable, and efficient middleware product independently developed by Alibaba Group to resolve scalability issues with single-host relational databases.
+With its compatibility with MySQL protocols and syntaxes, DRDS enables database/table sharding, smooth scaling, configuration upgrade/downgrade,
+transparent read/write splitting, and distributed transactions, providing O&M capabilities for distributed databases throughout their entire lifecycle.
```

Comment:
You can add a link in this docs and you can refer to: https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/website/docs/r/cen_instance.html.markdown